keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -40,12 +55,14 @@ def call(self, inputs):
         if self.norm_std:
             x = x / self._expand_non_channel_dims(self.norm_std, x)
 
-        return x
+        return x, y
```

Comment:
probably this should be `keras.utils.pack_x_y_sample_weight(x, y, sample_weight)`?

Reply:
This still needs doing I think (otherwise you toss sample_weight).

Reply:
Done